server/cloudformation/apps.go

```diff
@@ -0,0 +1,116 @@
+package cloudformation
+
+import (
+	"github.com/jinzhu/gorm"
+	"github.com/remind101/empire"
+	"golang.org/x/net/context"
+)
+
+type AppsProvisioner struct {
+	empire *empire.Empire
+}
+
+func (p *AppsProvisioner) Provision(req Request) (id string, data interface{}, err error) {
+	ctx := context.Background()
+	user := &empire.User{Name: "cloudformation"}
+
+	switch req.RequestType {
+	case Create:
+		name := req.ResourceProperties["Name"].(string)
+		app, err := p.empire.AppsFind(empire.AppsQuery{
+			Name: &name,
+		})
+		if err != nil && err != gorm.RecordNotFound {
+			return "", nil, err
+		}
+
+		app, err = p.empire.Create(ctx, empire.CreateOpts{
```

Comment:
this should account for empire requiring messages, maybe just supplying a default one?